src/documents/ChangeApplierAPI.md

```diff
@@ -8,9 +8,9 @@ This section explains and documents the various Javascript API calls for instant
 
 ## Registering interest in model changes using a ChangeApplier ##
 
-### Declarative style ###
+### Declarative style for listening to changes ###
 
-The declarative style for registering interest in change events uses an entry in the `modelListeners` options area of a `modelRelayComponent`. These listeners are attached to the applier during the construction process of the entire component (and its surrounding tree) and so will therefore become notified as part of the [initial transaction](ModelRelay.md#the-initial-transaction) - they will therefore get to observe the model changing state from its primordial value of undefined to holding their initial resolved value. This is the **recommended** way of listening to model changes using the ChangeApplier system.
+The declarative style for registering interest in change events uses an entry in the `modelListeners` options area of a `modelComponent`. These listeners are attached to the applier during the construction process of the entire component (and its surrounding tree) and so will therefore become notified as part of the [initial transaction](ModelRelay.md#the-initial-transaction) - they will therefore get to observe the model changing state from its primordial value of undefined to holding their initial resolved value. This is the **recommended** way of listening to model changes using the ChangeApplier system.
```

Comment:
"initial" already signifies the same as it did in previous versions of the framework - that is, the "initial synchronised value of the model" taking into account all sources of initial values. I'm open to improved suggestions for "primordial" - perhaps "grandeval" or "sephiric"?